spec/features/user_signs_up_spec.rb

```diff
@@ -24,6 +24,7 @@
 
     click_button "Sign up"
 
+    expect(find('img')['src']).to have_content "default.jpg"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.